### DISCUSSION TITLE
fc_script.exe dont working

### DISCUSSION THREAD

**Discussion by Acidcastle** [InoriRus/Kyty#83] [Q&A]
![۲۰۲۲۱۲۰۱_۱۵۴۷۰۵](https://user-images.githubusercontent.com/118968131/205870791-51e232fd-290c-469e-93ec-cfa09e281d8d.jpg)
![۲۰۲۲۱۲۰۱_۱۵۴۶۳۴](https://user-images.githubusercontent.com/118968131/205871358-631f9671-0861-4029-b15e-d2dee3405be2.jpg)
Hi fc_script.exe . does not work What is the problem?

**Comment by danni551** [comment]
I fixed it by updating my gpu driver
